refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+| `answers` | 0-* | Any of the terms the meaning of which is defined by the present vocabulary |
+| `message` | 0-1 | Optional string comment, motivation or explanation of Demand |
+| `lang` | 0-1 | Optional string Language of textual message associated with demands in the format of [FRC5646](https://datatracker.ietf.org/doc/rfc5646/) |
+| `includes` | 0-* | Optionally an array of one or more [Privacy Request Response](#privacy-request-response)s |
+| `data` | 0-* | Optionally concrete data to which access is being given (Format **TBD**) |
+
+A Privacy Request Response MUST have:
+- a unique ID,
+- a clear indication of the particular Privacy Request or the particular Demand to which its content refers,
+- a date,
+- a status.
+
+Privacy Request Responses can be nested. One can imagine a Privacy Request Response to a particular Privacy Request, that `includes` Privacy Request Responses to the particular Demands made in that Privacy Request. Several Systems MAY respond to the same Privacy Request or Demand, and one System MAY nest them in order to gather them and send them back to the Data Subject.
+
+When a Demand is being denied, the Privacy Request Response MUST provide a `motive`.
+
+>To illustrate the `answers` value, we can imagine a `TRANSPARENCY.DATA-CATEGORIES` Demand, to which a response may include `answers`: `CONTACT`, `IMAGE`. Or a `TRANSPARENCY.KNOWN` Demand to which the answer may include `answers`: `YES`.
+
+### Consent
+
+A Consent is given by one Data Subject which can be identified by one or more [Data Subject Identities](#decentralized-identity-of-data-subjects).
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+| `consent-id` | 1 | a string in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `date` | 1 | Date and Time when Consent was given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `expires` | 0-1 | Date and Time when Consent expires in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `target` | 0-1 | Optionally one of {`ORGANISATION`, `PARTNERS`, `SYSTEM`} to indicate the category of Systems to which consent for processing is given. In absence of indication `SYSTEM` is assumed. |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) in absence of which the Consent SHOULD be interpreted as unlimited |
+| `replaces` |  0-* | Optionally one or more 'consent-id's of previous consents that have became void when this consent was made |
+| `replaced-by` |  0-* | Optionally one or more 'consent-id's of previous consents that have became void when this consent was made |
+
+### Data Capture
```

Comment:
Are properties defined in the context of privacy requests? Or is this global definition of a `data capture` and `fragments` (capture system component)?

Reply:
@m4rk055  global definition